### ISSUE TITLE
Could you elaborate more on using pre-trained seq2seq model?

### ISSUE THREAD

**Issue by J-Miao** [varinf/TransformersDataAugmentation#3]
Thanks for your work, and I gained a lot of insight from it! 

When trying to implement your methods I have some issues on the seq2seq model, as there is very limited support for BART in existing toolkits and there are not many details in your paper about using BART.  Can you provide more information on the hyper-parameters for finetuning including number of epochs and learning rate? 

Also for generation, your paper says "beam search with a beam size of 5", but how do you generate the augmented data? The way I can think is to use the sample() method, but I cannot figure out more details.

**Comment by varunentropy** [comment]
Training: LR=1e-05; max epcoh 20; Used dev set to pick the best model.

Generation: I use fairseq-generate with the following params. Note that we implemented our own version of word and span masking based on BART original code. 

--replace-length 1 --mask-length {word/span} --mask 0.20 \
                        --batch-size 64 --beam 5 \
                        --no-repeat-ngram-size 2 \
                        --max-len-b 50 --prefix-size {2 for SST2, 3 for TREC and SNIPS} \
                        --gen-subset train

**Comment by varunentropy** [comment]
We open sourced the code. Please review the code and let me know if you still have a question. Thanks.
